R/LearnerAvg.R

```diff
@@ -45,20 +55,31 @@
 LearnerClassifAvg = R6Class("LearnerClassifAvg", inherit = LearnerClassif,
   public = list(
     initialize = function(id = "classif.avg") {
+      ps = ParamSet$new(params = list(
+        ParamUty$new("measure", custom_check = function(x)
+          check_r6(if (is.character(x)) msr(x) else x, "MeasureClassif"), tags = "train"),
```

Comment:
(1) Since the same checks are done both here and in the RegrAvg, maybe they should be functions in the package namespace.

(2) I don't think `msr(x)` needs to be `check_r6`'d. Also this throws an error if `x` is not in `msr()`, when it only should return TRUE or a character(1). This should probably be in the direction of
```r
if (is.character(x)) {
  if (x %nin% mlr_measures$keys()) "Is a `character` but not a known measure" else TRUE
} else {
  check_r6(x, "MeasureClassif")
}
```

Reply:
I factored them out.